### ISSUE TITLE
Cannot reproduce results, could you please provide the pretrained weights?

### ISSUE THREAD

**Issue by RafaelAZV** [hasanhuz/SpanEmo#8]
Hi! I am trying to reproduce the results of paper and while training, the best results I can get on dev set is 
Val_loss    F1-Ma.    F1-Mi         JS
 0.3648    0.4966    0.6917    0.5657

And this values are reached at the second epoch. Im using the joint loss as the paper suggests, with alpha = 0.2. Is there anything else i should do?

Could you please provide the pretrained weights?

Thanks in advance.

**Comment by hasanhuz** [comment]
Please, feel free to reopen this issue if you still have any questions!
